internal/yggdrasil/yggdrasil.go

```diff
@@ -184,7 +184,7 @@ func (h *Handler) PostDataMessageForDevice(ctx context.Context, params yggdrasil
 		if err != nil {
 			return operations.NewPostDataMessageForDeviceBadRequest()
 		}
-		logger.Info("received heartbeat", "content", heartbeat)
+		logger.V(1).Info("received heartbeat", "content", heartbeat)
```

Comment:
Why the increase of verbosity here?

Reply:
Based on https://github.com/jakub-dzon/k4e-operator#implementation-details, it decreases the verbosity
and from the documentation:
```
	// V returns an Logger value for a specific verbosity level, relative to
	// this Logger.  In other words, V values are additive.  V higher verbosity
	// level means a log message is less important.  It's illegal to pass a log
	// level less than zero.
	V(level int) Logger
```
Hence, we shouldn't see the details for those messages when the log level is set to `info`

Reply:
Ok, my bad; I confused zap logging priority with logr verbosity level.